airgun/entities/contenthost.py

```diff
@@ -41,6 +42,13 @@ def read(self, entity_name, widget_names=None):
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
         return view.read(widget_names=widget_names)
 
+    def get_details(self, entity_name, widget_names=None):
```

Comment:
As discussed please update the method name here, It will be more readable to know that this is related to older UI read

Reply:
> As discussed please update the method name here, It will be more readable to know that this is related to older UI read

@omkarkhatavkar Have already updated, thank you very much.